gitconfig

```diff
@@ -11,16 +11,19 @@
   ci = commit -v
   co = checkout
   st = status
+  graph = log --color --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset' --date=short --abbrev-commit
 [core]
   excludesfile = ~/.gitignore
   autocrlf = input
 [merge]
-  ff = only
+  ff = false
 [commit]
   template = ~/.gitmessage
 [fetch]
   prune = true
 [rebase]
   autosquash = true
+[branch]
+  autosetuprebase = always
```

Comment:
Is there never a case when rebase wouldn't be used? I imagine in branches that you don't own, this could be harmful.